### DISCUSSION TITLE
Animation Blink, What curve to add?

### DISCUSSION THREAD

**Discussion by sommerlost** [Sixze/ALS-Refactored#104] [Q&A]
I've got a basic emote setup that transitions from a grounded pose. Every thing work's great but I'm getting the single frame blink in the animation whenever the blend completes. Traditionally in ALS you would add a curve to the animation for "Base Pose N Curve", but that doesn't seem to exist in ALS Refactored. 

Do you mind explaining how you are blending full body animations cleanly without the frame blink? Thanks!

**Comment by Sixze** [comment]
I don't quite understand your issue, but the `BasePose_N` curve is now called `PoseStanding`.

**Reply by sommerlost** [reply]
You answered my question as far as clearing up the single frame "blink" on the character mesh. - I was just further asking if you had any insights into the best place a full body animation.

Currently I've transitions running back and forth from the Grounded state (Inside the Locomotion graph) to something simple like an Emote. I'm just triggering a bool from the character pawn and then evaluating the remaining time of the animation for the transition back. Inserting this interaction here seems to make sense, as the foot locking doesn't interfere. But I still can't figure out how to lock the character in place until the animation finishes. I'm also assuming this should be performed on the animation graph.

![ALS_where to add new states](https://user-images.githubusercontent.com/7546149/177919538-dac2bdd1-27c8-4079-ad48-346fafaabb69.png)


I've done this with ALS in the past, but that was also using montages instead of animation sequences. Which if I remember correctly can affect character movement different. Thanks!

**Reply by Sixze** [reply]
There is a `Locomotion.PostLocomotion` animation slot that is used for full body animation like rolling and mantling.

For movement block it is better to use some external gameplay system, like the **Gameplay Ability System**: create an ability that plays an emote montage and block character movement inside it until the ability ends.
